src/main/flight/pid.h

```diff
@@ -115,6 +115,10 @@ typedef struct pidProfile_s {
     uint8_t throttle_boost;                 // how much should throttle be boosted during transient changes 0-100, 100 adds 10x hpf filtered throttle
     uint8_t throttle_boost_cutoff;          // Which cutoff frequency to use for throttle boost. higher cutoffs keep the boost on for shorter. Specified in hz.
     uint8_t  iterm_rotation;                    // rotates iterm to translate world errors to local coordinate system
+    uint8_t iterm_relax_cutoff_low;        // Slowest setpoint response to prevent iterm accumulation
+    uint8_t iterm_relax_cutoff_high;       // Fastest setpoint response to prevent iterm accumulation
+    uint8_t iterm_relax;                   // Enable iterm suppression during stick input
```

Comment:
The idea here is to have presets that are sensible so only the feature needs to be turned on to start working. If we do what you suggest there's also an issue if only one cutoff is set which would essentially always disable the iterm.

Reply:
OK, that's reasonable argument.

Setting only one cutoff should work fine (if 0 means infinite frequency). With both cutoffs at +Inf Hz, it should behave as relax disabled, unless failsafe interferes

Reply:
@mikeller, I guess I was inspired by anti-gravity which you can switch on or off, but it doesn't lose your settings in the process. To me that seems useful. I find the definition of "a non zero number means the cutoff in hz, 0 means infinity and both 0 means feature off inconsistent and confusing. But if you think it's fits better with the overall config approach of betaflight I can certainly make that change.

Reply:
< doesn't really need to be checked. It doesn't matter which one is higher.

Reply:
@joelucid: Ah, so your plan is to change the code that setting one of the values to 0 means 'infinity'? Will setting both of them to infinity actually work? If not, having both 0 will require special handling, either by disabling the feature, or by resetting the (in this case invalid) config to defaults (off). And yes, while it is somewhat confusing, it is in line with how it is done in other places, like the notch filters.

Re the ordering not mattering, this is a bit confusing as well, and will lead to users wondering what happens / why it does not break when they have low cutoff > high cutoff. Would it make sense to use different names, like cutoff 1 / cutoff 2?

Reply:
@mikeller that's not my plan but it's what @ledvinap proposed and I thought you liked? But I can do it that way - I'm not that passionate either way. I do think low and high is easier to understand than 1 and 2 and if you expect user confusion due to it not breaking if they are used the other way around maybe that can indeed be prevented in the config tool?

Reply:
If @McGiverGim concern's rg YAW turn out to be unsolvable maybe we can have iterm_relax = [ALL, RP, OFF]. Might also need a separate two filter cutoffs for yaw due to lower pid authority on that axis.

Reply:
I wasn't so much commenting on adding the ability to set one side to 0 to make it go to infinity (but that's definitely an option if the resulting behaviour is useful), but on using 0 values to disable the feature.

Reply:
Ok. @mikeller lets wait for what comes out of the yaw issue before making a call. Just incidentally: I really dislike the way the notches are configured because I often don't remember what I had in there after switching one off. And there it's one number. Here you'll have to remember two numbers if you want to switch off and on.

Reply:
@joelucid: Agreed, if there will be multiple different modes for it, it needs a separate parameter to enable it. Re notch filters, these are not the only parameters were 0 is used as a special value to disable a feature, there's a dozen or so more. If we went and added extra parameters to enable / disable all of these, we'd probably have to remove some more features from most F3 builds to make space for this...